dss/config.py

```diff
@@ -416,9 +417,14 @@ def get_OIDC_email_claim():
 
     @staticmethod
     def get_authz_url():
-        if Config._AUTH_URL is None:
-            Config._AUTH_URL = Config._get_required_envvar("AUTH_URL")
-        return Config._AUTH_URL
+        if Config._CURRENT_CONFIG == BucketConfig.NORMAL:
```

Comment:
Should probably be:
```
        if (Config._CURRENT_CONFIG == BucketConfig.TEST or Config._CURRENT_CONFIG == BucketConfig.TEST_FIXTURE):
            host = "127.0.0.1"
            port = "54321"
            return f"http://{host}:{port}"
        else:
            if Config._AUTH_URL is None:
                Config._AUTH_URL = Config._get_required_envvar("AUTH_URL")
            return Config._AUTH_URL
```

Reply:
Another option:
```
if Config._CURRENT_CONFIG in [BucketConfig.TEST, BucketConfig.TEST_FIXTURE]:
    ...
```

Reply:
good suggestions all. went with the `if X in [list]` syntax